content/docs/application-mobility/troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Troubleshooting"
+linkTitle: "Troubleshooting"
+weight: 4
+Description: >
+  Troubleshooting guide
```

Comment:
This is inconsistent with other sections description, like "**Deployment for** Dell Container Storage Modules (CSM) for Application Mobility".

Reply:
Updated this and others to just the title of the page to remain consistent